src/app-gocardless/banks/commerzbank_cobadeff.js

```diff
@@ -0,0 +1,56 @@
+import Fallback from './integration-bank.js';
+import { formatPayeeName } from '../../util/payee-name.js';
+
+/** @type {import('./bank.interface.js').IBank} */
+export default {
+  ...Fallback,
+
+  institutionIds: ['COMMERZBANK_COBADEFF'],
+
+  accessValidForDays: 179,
+
```

Comment:
With https://github.com/actualbudget/actual-server/pull/551 in mind this property can be removed.

```suggestion
```